### PR TITLE
Removed takeover component and its related bits

### LINKED ISSUE #2442: Evaluate if we can remove "takeover" component and "upcoming" component
... since it's an oooooolllllldddddd component.

This includes `takeover.jsx`, `upcoming.jsx`, related SCSS, and some portions of `main.js`

### PR DESCRIPTION
**Pull request by mmmavis**
Closes #2442 

- Removed takeover component and its related SCSS and html bits
- Removed `SHOW_TAKEOVER` env var from various config files. Hopefully I didn't miss anything.
- Added check for `seen-takeover` cookie and remove it if it exists.

### CODE REVIEW

File: source/js/main.js [hunk 1]
```diff
@@ -221,25 +220,8 @@ let main = {
...
-        elWrapper.style.overflow = null;
-        elWrapper.style.height = null;
-      };
-
-      apps.push(new Promise(resolve => {
-        ReactDOM.render(<Takeover onHide={onTakeoverHide} whenLoaded={() => resolve()}/>, document.querySelector(`#view-home .takeover`));
-        Cookies.set(`seen-takeover`, `true`, { expires: 365 });
-        ReactGA.pageview(`/welcome-splash`);
-      }));
+    if (Cookies.get(`seen-takeover`)) {
```
Comment (mmmavis) [inline source/js/main.js]: do we need to do this?

Comment (patjouk) [reply source/js/main.js]: I would say no: @alanmoo, @Pomax can one of you confirm?

Comment (alanmoo) [reply source/js/main.js]: I...don't think so?

**Comment by mmmavis** [comment]
🎉

**Pull request merged**